### PR TITLE
Reg 64 add possibility for communication for all users

### PR DESCRIPTION
**Pull request by MarekBrandt**

### CODE REVIEW

File: lib/models/message.dart [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  Message(
+      {required this.type,
+      required this.receiverType,
+      this.teamId,
+      this.value});
+
+  Message.fromJson(Map<String, String> json)
+      : type = MessageType.values.byName(json['type']!),
+        receiverType = MessageReceiverType.values.byName(json['receiverType']!),
+        teamId = json['teamId'],
```
Comment (bartoz1) [inline lib/models/message.dart:16]: it will return null if such key is not defined, right?

File: lib/services/event_message_handler.dart [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    DatabaseReference messRef =
+        databaseReference.child('messages').child(eventId);
+
+    messageStream = messRef.onChildAdded.listen((event) {
+      final messageData = Map<String, String>.from(
+          event.snapshot.value as Map<Object?, Object?>);
+      final message = Message.fromJson(messageData);
+      print(message);
+
+      if (message.receiverType == MessageReceiverType.all ||
```
Comment (bartoz1) [inline lib/services/event_message_handler.dart:30]: that's a long if statement haha

**Review by bartoz1 — APPROVED** [review]

File: lib/models/message.dart (hunk 1 above)
Comment (MarekBrandt) [reply lib/models/message.dart:16]: Yes

File: lib/services/event_message_handler.dart (hunk 2 above)
Comment (MarekBrandt) [reply lib/services/event_message_handler.dart:30]: yep, I will try to make this better in next pr :D

**Pull request merged**